src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -21,31 +32,49 @@ const GovernanceSection = () => (
       </AccordionSummary>
 
       <AccordionDetails sx={({ spacing }) => ({ padding: `0 ${spacing(3)}` })}>
-        <WidgetBody>
-          <Grid spacing={3} container>
-            <Grid item xs={8}>
-              <Card>
-                <Box m={2} sx={{ minHeight: '200px' }}>
-                  <Typography variant="h3">
-                    <strong>Snapshot</strong>
-                  </Typography>
-                </Box>
-              </Card>
-            </Grid>
-            <Grid item xs={4}>
-              <Card>
-                <Box m={2} sx={{ minHeight: '200px' }}>
-                  <Typography variant="h3">
-                    <strong>Claiming app</strong>
-                  </Typography>
-                </Box>
-              </Card>
+        {fetchingSafeClaimingApp ? (
+          <Paper>
+            <Box sx={{ height: '300px' }} display="flex" alignItems="center" justifyContent="center">
+              <Typography variant="h1" color="text.secondary">
+                Loading Safe Claiming App...
```

Comment:
I know that they both hypothetically the Claiming App but I think it would look visually better if we split it into the same layout as the widgets, e.g. [ Loading Snapshot ] [ Loading Claiming ]. What do you think? cc @liliiaorlenko

Reply:
In this case we are just fetching the remote Safe apps endpoint. Each of the widgets loads with its own Skeleton component. So this "loading" component was more for the whole Governance section

Reply:
I would adjust it to reference "governance" instead of the Claiming App specifically then.

Reply:
That's quite generic imo. If you wanted to specify apps, I would say splitting the loading section in two as my suggestion further up would make sense.